docs/cloud/pouta/heat-orchestration.md

```diff
@@ -64,45 +71,167 @@ stacks:
     | 98077bd5-9d69-47c3-98db-b0e19a60b1fa | my-heat-stack | CREATE_COMPLETE | 2016-06-08T07:34:46Z | None         |
     +--------------------------------------+---------------+-----------------+----------------------+--------------+
 
-### Heat guidelines and command references
+### Explanations
+A template is composed of two major sections: 
+- The version used (`heat_template_version`)
+- The resource(s) (`resources`)
 
-For more information, visit the OpenStack
-[Heat wiki]. For a full reference
-to the OpenStack command line client, see the [command line
-reference], and for the Heat command
-line client, see the [Heat CLI reference].
+There are also optional sections like:
+- The description (`description`)
+- The parameter group(s) (`parameter_groups`)
+- The parameter(s) (`parameters`)
+- The output(s) (`outputs`)
+- The condition(s) (`conditions`)
 
-To try out the example provided here, modify at least the "key\_name"
-in the template code:
+Here is a detail of each sections:  
 
-    heat_template_version: 2015-10-15
+`heat_template_version`  
+&nbsp;&nbsp;&nbsp;&nbsp; Indicates that the YAML document is a HOT template of the specified version.
 
-    description: >
-      Simple template to deploy
-      a single instance in cPouta
+`description`  
+&nbsp;&nbsp;&nbsp;&nbsp; Allows for giving a description of the template.
 
-    resources:
-      instance0:
-        type: OS::Nova::Server
-        properties:
-          image: Ubuntu-16.04
-          flavor: standard.tiny
-          key_name: my-key
-      instance1:
-        type: OS::Nova::Server
+`parameter_groups`  
+&nbsp;&nbsp;&nbsp;&nbsp; Allows for specifying how the input parameter should be grouped and the order to provide the parameters.
+
+`parameters`  
+&nbsp;&nbsp;&nbsp;&nbsp; Allows for specifying input parameters that have to be provided when instantiating the template.
+
+`resources`  
+&nbsp;&nbsp;&nbsp;&nbsp; Contains the declaration of the single resources of the template.
+
+`outputs`  
+&nbsp;&nbsp;&nbsp;&nbsp; Allows for specifying output parameters available once the template has been instantiated.
+
+`conditions`  
+&nbsp;&nbsp;&nbsp;&nbsp; Includes statements which can be used to restrict when a resource is created or when a property is defined
+
+
+### Advanced example: create a template to build one or more instances
+The plan here is:
```

Comment:
This might need an extra empty line. Now the list is rendered in a single line.

Reply:
Fixed